catalog-info.yaml

```
@@ -0,0 +1,18 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'edx-drf-extensions'
+  description: "This library includes various cross-cutting concerns related to APIs."
```

Are you ok with the following suggestion? Nimisha always wished this were called `edx-rest-api-provider` for consistency with `edx-rest-api-client`.
```suggestion
  description: "This library includes various cross-cutting concerns for providing APIs."
```